management/usage.md

```diff
@@ -2,14 +2,24 @@
 
 This folder contains all the management with the database directly, or, possibly later on, with the self-built API. You can use it in any Python file that is connected with the bot.
 
+## Jump
+[init.py](#init)  
+[db.py](#db)  
+-> [Raw commands](#raw)  
+-> [Game-related data](#game)  
+-> [Kill queue](#kill)  
+-> [Channel commands](#channel)  
+-> [Other commands](#other)  
+[position.py](#position)  
+
 ## Relevant files
 
 The folder contains a few files. Here's an explanation of what everything is and what it all does.
 
-### __init__.py
+### <a head="#init"></a>init.py
```

Comment:
wtf is this? use regular `id="init"`?

Reply:
Does that matter? Writing __init__.py only makes the text bold, which it already is as it's a title. So yeah.

It'd be cool to have a way to filter it out, but it's not THAT crucial. It shouldn't be a road block to understand how the folder works.